### PR TITLE
[#1294] 404 refresh

### LINKED ISSUE #1294: Add a 404 page matching akvo.org's
Update the 404 page to look like Akvo.org's.
screen shot 2015-03-03 at 16 51 25

### PR DESCRIPTION
**Pull request by damienallen**
Fixes #1294

### CODE REVIEW

**Comment by damienallen** [comment]
Test plan:

Open invalid URL and compared 404 page to those on the main Akvo website.

File: akvo/templates/404.html [hunk 1]
```diff
@@ -5,9 +5,15 @@
...
+    <h1 class="text-center" style="margin-top: 1em; margin-bottom: 1em;">{% trans '404 Error Page. Not Found!' %}</h1>
     <div class="row">
-        <h1>{% trans '404 Error Page. Not Found!' %}</h1>
-        <h2><small>{% trans 'You might want to try our home page, or the menu bar above' %}</small></h2>
+        <div class="col-md-6"><img src="/static/images/404_surprise.png" alt="Surprise" class="center-block" style="max-height: 450px;" title=""></div>
+        <div class="col-md-6">
+        <p style="font-size:150%; margin-top:15%; margin-bottom:10%;">
+            Or isn't this what you were hoping for?<br><br>
+            In that case something's gone wrong.<br><br>
+            You might want to try our home page, or the menu bar above.</p>
```
Comment (KasperBrandt) [inline akvo/templates/404.html]: @damienallen Could you add a ```{% trans %}``` tag around these lines as well?

File: akvo/templates/404.html [hunk 2]
```diff
@@ -10,9 +10,9 @@ <h1 class="text-center" style="margin-top: 1em; margin-bottom: 1em;">{% trans '4
         <div class="col-md-6"><img src="/static/images/404_surprise.png" alt="Surprise" class="center-block" style="max-height: 450px;" title=""></div>
         <div class="col-md-6">
         <p style="font-size:150%; margin-top:15%; margin-bottom:10%;">
-            Or isn't this what you were hoping for?<br><br>
-            In that case something's gone wrong.<br><br>
-            You might want to try our home page, or the menu bar above.</p>
+            {% trans Or isn't this what you were hoping for? %}<br><br>
+            {% trans In that case something's gone wrong. %}<br><br>
+            {% trans You might want to try our home page, or the menu bar above. %}</p>
```
Comment (KasperBrandt) [inline akvo/templates/404.html]: @damienallen It is now broken 😉 .. The ```{% trans %}``` tags need  ```"```'s for the strings.

**Comment by KasperBrandt** [comment]
Code checked 👍

**Pull request merged**